internal/openshift/controller.go

```diff
@@ -288,3 +306,12 @@ func sendUserToIdler(idler *idler.UserIdler, user model.User) {
 			"Unable to send user to channel. Discarding event.")
 	}
 }
+
+// checks weather user name is present in Disabled Map.
+// if present return Disabled/true.
+func (c *controllerImpl) isUserDisabled(user string) Status {
```

Comment:
shouldn't this function be in DisabledUsers instead?

Reply:
Done with new approach.